src/Paket.Core/PublicAPI.fs

```diff
@@ -165,10 +165,10 @@ type Dependencies(dependenciesFileName: string) =
                                               projectName, installAfter, runResolver))
 
     /// Adds credentials for a Nuget feed
-    member this.AddCredentials(source: string, username: string, password : string, authType : string) : unit =
+    member this.AddCredentials(source: string, username: string, password : string, authType : string, verify : bool) : unit =
```

Comment:
this one is in the PublicAPI and would be breaking.
We need a second function and keep the old one

Reply:
Done